components/brave_rewards/common/pref_names.cc

```diff
@@ -88,6 +88,8 @@ const char kWalletBitflyer[] = "brave.rewards.wallets.bitflyer";
 const char kWalletGemini[] = "brave.rewards.wallets.gemini";
 const char kWalletCreationEnvironment[] =
     "brave.rewards.wallet_creation_environment";
+const char kAdsWereDisabled[] = "brave.brave_ads.were_disabled";
```

Comment:
I thought I'd move them over to rewards since there are only usages of the prefs in rewards (I suppose these prefs were misplaced originally), and we'll be gutting the prefs pretty soon anyway. If you feel strongly about it, I can move them back

Reply:
If they will be remove entirely soon anyway, that seems fine.